### PR TITLE
Update to Stitches beta pre-release

### PR DESCRIPTION
**Pull request by peduarte**

### CODE REVIEW

**Comment by colmtuite** [comment]
The font sizes are all messed up in this branch. It looks like every text component is a size smaller than what it should be?

File: components/Button.tsx [hunk 1]
```diff
@@ -167,41 +194,56 @@ const StyledButton = styled(DEFAULT_TAG, {
...
+type ButtonVariants = Omit<StitchesVariants<typeof StyledButton>, 'size'>;
+type ButtonOwnProps = ButtonCSSProp & ButtonVariants & { size?: any };
+
 type ButtonComponent = Polymorphic.ForwardRefComponent<typeof DEFAULT_TAG, ButtonOwnProps>;
 
 export const Button = React.forwardRef((props, forwardedRef) => {
   return <StyledButton {...props} ref={forwardedRef} />;
 }) as ButtonComponent;
+
+Button.toString = () => `.${StyledButton.className}`;
```
Comment (vladmoroz) [inline components/Button.tsx:249]: Why/when is this needed?

File: components/Card.tsx [hunk 2]
```diff
@@ -55,23 +57,26 @@ const StyledCard = styled(DEFAULT_TAG, {
...
+            backgroundColor: '$panel',
+            transform: 'translateY(-2px)',
+            '&::before': {
+              opacity: '1',
+            },
           },
         },
         '&:active': {
           transform: 'translateY(0)',
+          transition: 'none',
```
Comment (vladmoroz) [inline components/Card.tsx:71]: I love this `transition: none` style for the pressed state

File: components/Flex.tsx [hunk 3]
```diff
@@ -126,7 +126,7 @@ export const Flex = (React.forwardRef<HTMLDivElement, Props>(
               flexDirection,
               // && for Injection oooordeeerrr
               // since atomic stitches has issues with guaranteeing the order of injection
-              '&& > *': {
+              '& > *': {
```
Comment (vladmoroz) [inline components/Flex.tsx:129]: Looks like the comment is now obsolette

File: components/Button.tsx (hunk 1 above)
Comment (peduarte) [reply components/Button.tsx:249]: This is needed if you need to reference a "non-stitches" component in the Style Object. The `ControlGroup` does

**Review by vladmoroz — APPROVED** [review]

**Review by colmtuite — APPROVED** [review]

**Pull request merged**